gips/0082.md

```diff
@@ -224,6 +224,9 @@ This command allows providers to register their service, specifying their operat
  - [DataEdge addresses](https://github.com/graphprotocol/contracts/blob/main/packages/data-edge/addresses.js)
 - The relevant contract sources are `DataEdge` ([DataEdge.sol](https://github.com/graphprotocol/contracts/blob/main/packages/data-edge/contracts/DataEdge.sol)) and `EventfulDataEdge` ([EventfulDataEdge.sol](https://github.com/graphprotocol/contracts/blob/main/packages/data-edge/contracts/EventfulDataEdge.sol)). `EventfulDataEdge` emits an event upon registration, while `DataEdge` does not.
 
+# Meaning and commitment
+
+Approving this GIP settles affirmatively the question of whether Substreams is "On The Network".
```

Comment:
I don't think we need to include this on the GIP :) This "on the network" refers to some internal conversations amongst core devs - I don't think this is useful for the broader ecosystem.

Reply:
done